snuba/request/request_settings.py

```diff
@@ -132,6 +150,12 @@ def get_dry_run(self) -> bool:
     def get_legacy(self) -> bool:
         return False
 
+    def get_team(self) -> str:
+        return "subscription"
+
+    def get_feature(self) -> str:
+        return "subscription"
+
```

Comment:
I'm not sure how I feel about this being hardcoded. Can it not be populated further up the chain?

Reply:
For subscriptions? Can I ask why you don't like this?

Reply:
Are all subscriptions always the same? Can't you subscribe to different things?

Reply:
OK I think I see. Right now we lump all these queries into one thing. Every subscription even has the same referrer. Since one team owns subscriptions, and they are part of a single feature, we hardcoded things this way. I can move this logic up a level so that if we want to start separate metrics/release health alerts etc. we can do that.

Reply:
Yeah I would prefer that